src/MongoDB.Driver/MongoClient.cs

```diff
@@ -35,6 +35,22 @@ namespace MongoDB.Driver
     /// <inheritdoc/>
     public class MongoClient : MongoClientBase
     {
+        #region static
+        // private static methods
+        private static IEnumerable<ServerDescription> SelectServersThatDetermineWhetherSessionsAreSupported(ClusterConnectionMode connectionMode, IEnumerable<ServerDescription> servers)
```

Comment:
Same method as ClusterDescription.SelectServersThatDetermineWhetherSessionsAreSupported.

But copy/pasted here since we can't make public API changes this time around.